Cannot build caddy

### Info:
- Docker version (`$ docker --version`): Docker version 18.09.2, build 6247962
- Laradock commit (`$ git rev-parse HEAD`):  58d7d4fa0bfc6bf8980d1553a72c78aa8c097897
- System info (Mac, PC, Linux):  Ubuntu 18
- System info disto/version: 

### Issue:
a bunch of 'cannot find package" errors
_____

### Expected behavior:
just build caddy successfully
_____

### Reproduce:

Clone newest laradock,

docker-compose build caddy
_____

### Relevant Code:

```
Building caddy
Step 1/9 : FROM golang:alpine
 ---> 84aad42b0941
Step 2/9 : LABEL maintainer="Huadong Zuo <admin@zuohuadong.cn>"
 ---> Using cache
 ---> 57f1b2602cf9
Step 3/9 : RUN apk add --no-cache     openssh     git     build-base     && mkdir -p $GOPATH/src/golang.org/x/     && cd $GOPATH/src/golang.org/x/     && git clone https://github.com/golang/sys.git sys     && go get github.com/abiosoft/caddyplug/caddyplug     && caddyplug install-caddy     apk del build-base
 ---> Running in b7548cffd783
fetch http://dl-cdn.alpinelinux.org/alpine/v3.9/main/x86_64/APKINDEX.tar.gz
fetch http://dl-cdn.alpinelinux.org/alpine/v3.9/community/x86_64/APKINDEX.tar.gz
(1/34) Installing binutils (2.31.1-r2)
(2/34) Installing libmagic (5.35-r0)
(3/34) Installing file (5.35-r0)
(4/34) Installing gmp (6.1.2-r1)
(5/34) Installing isl (0.18-r0)
(6/34) Installing libgomp (8.2.0-r2)
(7/34) Installing libatomic (8.2.0-r2)
(8/34) Installing libgcc (8.2.0-r2)
(9/34) Installing mpfr3 (3.1.5-r1)
(10/34) Installing mpc1 (1.0.3-r1)
(11/34) Installing libstdc++ (8.2.0-r2)
(12/34) Installing gcc (8.2.0-r2)
(13/34) Installing musl-dev (1.1.20-r4)
(14/34) Installing libc-dev (0.7.1-r0)
(15/34) Installing g++ (8.2.0-r2)
(16/34) Installing make (4.2.1-r2)
(17/34) Installing fortify-headers (1.0-r0)
(18/34) Installing build-base (0.5-r1)
(19/34) Installing nghttp2-libs (1.35.1-r0)
(20/34) Installing libssh2 (1.8.2-r0)
(21/34) Installing libcurl (7.64.0-r1)
(22/34) Installing expat (2.2.6-r0)
(23/34) Installing pcre2 (10.32-r1)
(24/34) Installing git (2.20.1-r0)
(25/34) Installing openssh-keygen (7.9_p1-r4)
(26/34) Installing ncurses-terminfo-base (6.1_p20190105-r0)
(27/34) Installing ncurses-terminfo (6.1_p20190105-r0)
(28/34) Installing ncurses-libs (6.1_p20190105-r0)
(29/34) Installing libedit (20181209.3.1-r0)
(30/34) Installing openssh-client (7.9_p1-r4)
(31/34) Installing openssh-sftp-server (7.9_p1-r4)
(32/34) Installing openssh-server-common (7.9_p1-r4)
(33/34) Installing openssh-server (7.9_p1-r4)
(34/34) Installing openssh (7.9_p1-r4)
Executing busybox-1.29.3-r10.trigger
OK: 190 MiB in 49 packages
Cloning into 'sys'...
fetching missing dependencies: caddy, dnsproviders, hook.pluginloader
github.com/flynn/go-shlex
github.com/google/uuid
github.com/mholt/caddy/telemetry
github.com/mholt/caddy/caddyfile
github.com/mholt/caddy
Cloning into '/root/lib/caddy/gopath/src/github.com/caddyserver/dnsproviders'...
github.com/abiosoft/caddyplug (download)
github.com/abiosoft/caddyplug/internal
github.com/abiosoft/caddyplug
done fetching depedencies.

installing Caddy...
/root/lib/caddy/gopath/src/github.com/mholt/caddy/caddyhttp/browse/browse.go:32:2: cannot find package "github.com/dustin/go-humanize" in any of:
        /usr/local/go/src/github.com/dustin/go-humanize (from $GOROOT)
        /root/lib/caddy/gopath/src/github.com/dustin/go-humanize (from $GOPATH)
/root/lib/caddy/gopath/src/github.com/mholt/caddy/caddytls/config.go:27:2: cannot find package "github.com/go-acme/lego/certcrypto" in any of:
        /usr/local/go/src/github.com/go-acme/lego/certcrypto (from $GOROOT)
        /root/lib/caddy/gopath/src/github.com/go-acme/lego/certcrypto (from $GOPATH)
/root/lib/caddy/gopath/src/github.com/mholt/caddy/caddytls/tls.go:32:2: cannot find package "github.com/go-acme/lego/challenge" in any of:
        /usr/local/go/src/github.com/go-acme/lego/challenge (from $GOROOT)
        /root/lib/caddy/gopath/src/github.com/go-acme/lego/challenge (from $GOPATH)
/root/lib/caddy/gopath/src/github.com/mholt/caddy/caddytls/config.go:25:2: cannot find package "github.com/go-acme/lego/challenge/tlsalpn01" in any of:
        /usr/local/go/src/github.com/go-acme/lego/challenge/tlsalpn01 (from $GOROOT)
        /root/lib/caddy/gopath/src/github.com/go-acme/lego/challenge/tlsalpn01 (from $GOPATH)
/root/lib/caddy/gopath/src/github.com/mholt/caddy/caddyhttp/websocket/websocket.go:31:2: cannot find package "github.com/gorilla/websocket" in any of:
        /usr/local/go/src/github.com/gorilla/websocket (from $GOROOT)
        /root/lib/caddy/gopath/src/github.com/gorilla/websocket (from $GOPATH)
/root/lib/caddy/gopath/src/github.com/mholt/caddy/caddyhttp/httpserver/logger.go:26:2: cannot find package "github.com/hashicorp/go-syslog" in any of:
        /usr/local/go/src/github.com/hashicorp/go-syslog (from $GOROOT)
        /root/lib/caddy/gopath/src/github.com/hashicorp/go-syslog (from $GOPATH)
/root/lib/caddy/gopath/src/github.com/mholt/caddy/caddyhttp/basicauth/basicauth.go:35:2: cannot find package "github.com/jimstudt/http-authentication/basic" in any of:
        /usr/local/go/src/github.com/jimstudt/http-authentication/basic (from $GOROOT)
        /root/lib/caddy/gopath/src/github.com/jimstudt/http-authentication/basic (from $GOPATH)
/root/lib/caddy/gopath/src/github.com/mholt/caddy/caddy/caddymain/run.go:32:2: cannot find package "github.com/klauspost/cpuid" in any of:
        /usr/local/go/src/github.com/klauspost/cpuid (from $GOROOT)
        /root/lib/caddy/gopath/src/github.com/klauspost/cpuid (from $GOPATH)
/root/lib/caddy/gopath/src/github.com/mholt/caddy/caddyhttp/proxy/reverseproxy.go:43:2: cannot find package "github.com/lucas-clemente/quic-go" in any of:
        /usr/local/go/src/github.com/lucas-clemente/quic-go (from $GOROOT)
        /root/lib/caddy/gopath/src/github.com/lucas-clemente/quic-go (from $GOPATH)
/root/lib/caddy/gopath/src/github.com/mholt/caddy/caddyhttp/httpserver/server.go:34:2: cannot find package "github.com/lucas-clemente/quic-go/h2quic" in any of:
        /usr/local/go/src/github.com/lucas-clemente/quic-go/h2quic (from $GOROOT)
        /root/lib/caddy/gopath/src/github.com/lucas-clemente/quic-go/h2quic (from $GOPATH)
/root/lib/caddy/gopath/src/github.com/mholt/caddy/caddytls/config.go:30:2: cannot find package "github.com/mholt/certmagic" in any of:
        /usr/local/go/src/github.com/mholt/certmagic (from $GOROOT)
        /root/lib/caddy/gopath/src/github.com/mholt/certmagic (from $GOPATH)
/root/lib/caddy/gopath/src/github.com/mholt/caddy/caddyhttp/markdown/metadata/metadata_toml.go:20:2: cannot find package "github.com/naoina/toml" in any of:
        /usr/local/go/src/github.com/naoina/toml (from $GOROOT)
        /root/lib/caddy/gopath/src/github.com/naoina/toml (from $GOPATH)
/root/lib/caddy/gopath/src/github.com/mholt/caddy/caddyhttp/httpserver/tplcontext.go:35:2: cannot find package "github.com/russross/blackfriday" in any of:
        /usr/local/go/src/github.com/russross/blackfriday (from $GOROOT)
        /root/lib/caddy/gopath/src/github.com/russross/blackfriday (from $GOPATH)
/root/lib/caddy/gopath/src/github.com/mholt/caddy/caddyhttp/proxy/reverseproxy.go:41:2: cannot find package "golang.org/x/net/http2" in any of:
        /usr/local/go/src/golang.org/x/net/http2 (from $GOROOT)
        /root/lib/caddy/gopath/src/golang.org/x/net/http2 (from $GOPATH)
/root/lib/caddy/gopath/src/github.com/mholt/caddy/caddyhttp/httpserver/roller.go:23:2: cannot find package "gopkg.in/natefinch/lumberjack.v2" in any of:
        /usr/local/go/src/gopkg.in/natefinch/lumberjack.v2 (from $GOROOT)
        /root/lib/caddy/gopath/src/gopkg.in/natefinch/lumberjack.v2 (from $GOPATH)
/root/lib/caddy/gopath/src/github.com/mholt/caddy/caddyhttp/markdown/metadata/metadata_yaml.go:20:2: cannot find package "gopkg.in/yaml.v2" in any of:
        /usr/local/go/src/gopkg.in/yaml.v2 (from $GOROOT)
        /root/lib/caddy/gopath/src/gopkg.in/yaml.v2 (from $GOPATH)
exit status 1
ERROR: Service 'caddy' failed to build: The command '/bin/sh -c apk add --no-cache     openssh     git     build-base     && mkdir -p $GOPATH/src/golang.org/x/     && cd $GOPATH/src/golang.org/x/     && git clone https://github.com/golang/sys.git sys     && go get github.com/abiosoft/caddyplug/caddyplug     && caddyplug install-caddy     apk del build-base' returned a non-zero code: 1

```

--- a/caddy/Dockerfile
+++ b/caddy/Dockerfile
@@ -1,30 +1,5 @@
-FROM golang:alpine
+FROM abiosoft/caddy:no-stats
 
-LABEL maintainer="Huadong Zuo <admin@zuohuadong.cn>"
+CMD ["--conf", "/etc/caddy/Caddyfile", "--log", "stdout", "--agree=true"]
 
-RUN apk add --no-cache \
-    openssh \
-    git \
-    build-base \
-    && mkdir -p $GOPATH/src/golang.org/x/ \
-    && cd $GOPATH/src/golang.org/x/ \
-    && git clone https://github.com/golang/sys.git sys \
-    && go get github.com/abiosoft/caddyplug/caddyplug \
-    && caddyplug install-caddy \
-    apk del build-base
-
-ARG plugins="cors"
-
-## ARG plugins="cors cgi cloudflare azure linode"
-
-RUN caddyplug install ${plugins}
-
-RUN apk add --no-cache inotify-tools \
-    && echo -e "#!/bin/sh\nwhile inotifywait -e modify /etc/caddy; do\n\tpkill caddy\ndone " >> /start.sh \
-    && chmod +x /start.sh
-
-EXPOSE 80 443
-
-WORKDIR /var/www/public
-
-CMD ["sh","-c","/start.sh & /usr/bin/caddy -conf /etc/caddy/Caddyfile -agree"]
+EXPOSE 80 443 2015

